### PR TITLE
Add Mangalink

### PR DESCRIPTION
**Pull request by MikeZeDev**
Little captcha to pass. https://github.com/manga-download/haruneko/issues/373

First time we are stuck until timeout, then everything works. Minor inconvenience .

### CODE REVIEW

File: web/src/engine/websites/MangaLink.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { Tags } from '../Tags';
+import icon from './MangaLink.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as Madara from './decorators/WordPressMadara';
+import * as Common from './decorators/Common';
+
```
Comment (ronny1982) [inline web/src/engine/websites/MangaLink.ts:6]: Avoid adding website specific anti-scraping detections into the general `AntiScrapingDetection`.
Instead, place it directly into the related file for the website:
```typescript
AddAntiScrapingDetection(async (dom) => {
    return dom.querySelector('form#lsrecaptcha-form[action*="/.lsrecap/recaptcha?"]') ? FetchRedirection.Interactive : undefined;
});
```

File: web/src/engine/websites/MangaLink_e2e.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { TestFixture } from '../../../test/WebsitesFixture';
+
+//THIS test may fail because of recaptcha
```
Comment (ronny1982) [inline web/src/engine/websites/MangaLink_e2e.ts:3]: Consider to block comment the parts of the test that do not working like in [NeoxScan_e2e.ts](https://github.com/manga-download/haruneko/blob/master/web/src/engine/websites/NeoxScan_e2e.ts)

**Review by ronny1982 — CHANGES_REQUESTED** [review]
Should at least move anti-scraping detection to website file

File: web/src/engine/websites/MangaLink.ts (hunk 1 above)
Comment (MikeZeDev) [reply web/src/engine/websites/MangaLink.ts:6]: Oh, ok,

**Comment by MikeZeDev** [comment]
Removed captcha detection from new included js.

File: web/src/engine/AntiScrapingDetection.js [hunk 3]
```diff
@@ -1,2 +1,2 @@
 /* eslint-disable -- @preserved */
```
Comment (ronny1982) [inline web/src/engine/AntiScrapingDetection.js:1]: Probably no longer need to submit this file as there are no changes

**Review by ronny1982 — APPROVED** [review]

**Pull request merged**